pkgs/by-name/hy/hyprland-activewindow/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-activewindow";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "FieldOfClay";
+    repo = pname;
```

```suggestion
    repo = "hyprland-activewindow";
```
Avoid `repo = pname`